movie_api/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for move_api project.
```

Comment:
There is a typo in the project name. It should be 'movie_api' instead of 'move_api'.